src/utils/get-html-from-native-paste.js

```diff
@@ -0,0 +1,49 @@
+import { findDOMNode } from 'react-dom'
+
+/**
+ * Get clipboard html data by capturing the html inserted by the browser's native paste action.
```

Comment:
For all comments (including this one): Can you capitalize `HTML`, and make them formatted as paragraphs instead of individual line breaks, and always have punctuation at the end of the sentences.